Makefile

```diff
@@ -136,23 +136,17 @@ check: lib tests
 test: check
 
 ifndef WINDOWS
-# note: for coverage testing, build with:
-# make CFLAGS='--coverage -g3 -O0'
+# Note: for coverage testing, build with:
+#     make CFLAGS='--coverage -g3 -O0' LDFLAGS='--coverage -g3 -O0'
```

Comment:
Does `LDFLAGS` need `-g3 -O0`?

Reply:
Need: no. Benefit from: `-O` would just be ignored when not doing LTO. I'm not sure if `-g` makes a difference, but I confirm that it isn't needed to get coverage measurements.